forte/mrdsrg-spin-integrated/master_mrdsrg.cc

```diff
@@ -450,8 +450,8 @@ void MASTER_DSRG::init_dm_ints() {
         dm_[i] = BTF_->build(tensor_type_, "Dipole " + dm_dirs_[i], spin_cases({"gg"}));
     }
 
-    std::vector<psi::SharedMatrix> dm_a = ints_->mo_dipole_ints(true, true);
-    std::vector<psi::SharedMatrix> dm_b = ints_->mo_dipole_ints(false, true);
+    std::vector<psi::SharedMatrix> dm_a = ints_->mo_dipole_ints();
+    std::vector<psi::SharedMatrix> dm_b = dm_a;
```

Comment:
Can you please a comment about the logic here?

Reply:
Good catch. Because the MO one-body integrals are transformed using only `Ca` so that there is only one set of integrals.